### PR TITLE
Added unit test cases for below functionality:

### PR DESCRIPTION
**Pull request by sujata-m**
This PR contains - 

1. Updated all unit test cases
2. Unit test cases no longer require .env file
3. Fixed test harness issue.
4. Updated test folder structure.
5. Updated Readme file

### CODE REVIEW

**Review by sureshgaussian — CHANGES_REQUESTED** [review]
There is no update to readme.md file that gives me information about how to run these new tests. I guessed the process on how to run, but was not able to run even one. Please update readme.md file on how to run the unit tests.

File: .github/workflows/unit_test.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          pip install -r requirements.txt
+          pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple tdei-gtfs-csv-validator==0.0.31
+      - name: Creating Env file
+        run: |
+          touch .env
+          echo PROVIDER=Azure >> .env
+          echo UPLOAD_TOPIC=test_subscription_topic >> .env
+          echo VALIDATION_TOPIC=test_subscription_topic >> .env
+          echo UPLOAD_SUBSCRIPTION=test_subscription >> .env
+          echo STORAGECONNECTION='DefaultEndpointsProtocol=https;AccountName=tdeisamplestorage;AccountKey=l9JSJCGq9NrXqRVKApSk1wwV27aWaOVuxeY0NWOZz2svIlJzyncr3UFTzLoAanFbmJIeb2WmwIcS+AStj5gELg==;EndpointSuffix=core.windows.net' >> .env
```
[the remark this answers is not in the document]
Comment (sujata-m) [reply .github/workflows/unit_test.yaml]: @MashB told to remove this pipeline, removing the pipeline

File: README.md [hunk 2]
```diff
@@ -76,6 +76,20 @@ Add the new set of test inside `test/tests.json` file like -
 
 ```
 
+#### How to run the tests
+1. Set up the python env locally, see [here](#how-to-setup-aand-build)
```
Comment (susrisha) [inline README.md:80]: typo I think. it should be `how-to-setup-and-build` (There are two `a` in the above

**Review by susrisha — CHANGES_REQUESTED** [review]
one minor change in README

**Review by sureshgaussian — APPROVED** [review]
I am approving this PR. I ran the tests and looked at the report. It is fine. However, we need some changes, which we can pick up as a clean up task. I will file a devboard item for the improvements. https://dev.azure.com/TDEI-UW/TDEI/_workitems/edit/358/

**Pull request merged**